src/onboarding2/Welcome/OpenOrg.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { BackButton, Bar, Box, Button, GU, useTheme } from '@aragon/ui'
+import DomainField, {
+  DOMAIN_CHECK,
+} from '../../components/DomainField/DomainField'
+
+function OpenOrg({ onBack }) {
+  const theme = useTheme()
+  return (
+    <Box padding={5 * GU}>
+      <Bar
+        css={`
+          margin: -${5 * GU}px -${5 * GU}px 0;
+          border: 0;
+          border-bottom: 1px solid ${theme.border};
+          border-bottom-left-radius: 0;
+          border-bottom-right-radius: 0;
+        `}
+      >
+        <BackButton onClick={onBack} />
+      </Bar>
+
+      <div
+        css={`
+          display: flex;
+          flex-direction: column;
+          width: 100%;
+          height: ${36 * GU}px;
```

Comment:
@dizzypaty Perhaps we can decrease this height a bit, to make its proportions more rectangular than square (the width we're using is smaller than the design's by ~100px).